bindings/impex/xcs_catalog_enfinity_7.1.xjb

```diff
@@ -0,0 +1,17 @@
+<jxb:bindings version="3.0"
+    xmlns:jxb="https://jakarta.ee/xml/ns/jaxb"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+>
+    <jxb:globalBindings generateIsSetMethod="true" />
+
+    <jxb:bindings schemaLocation="../../schemas/xml/ns/enfinity/7.1/xcs/impex/catalog.xsd" node="/xs:schema">
+        <jxb:schemaBindings>
+            <jxb:package name="com.intershop.xsd.enfinity.v7_1.xcs.catalog" />
```

Comment:
The `v7_1` should be part of the package name? Does the catalog binding currently contain one / would it require migration efforts?